Simplenote/src/main/java/com/automattic/simplenote/Simplenote.java

```diff
@@ -242,36 +241,35 @@ public String getUserEmail() {
         return user != null ? user.getEmail() : null;
     }
 
-    private void checkReviewAccountOrVerifyEmail(final Activity activity) {
-        if (isFirstLaunch() || !NetworkUtils.isNetworkAvailable(this) || mHasShownReviewOrVerify) {
-            // Show nothing on first launch, no network connection, or has already shown review/verify.
-            return;
-        }
+    public AccountVerificationWatcher getReviewAccountVerifier() {
+        return mAccountVerificationWatcher;
+    }
```

Comment:
this function is never called, right? we don't need it?

Reply:
Yes, it is dead code. Fixed in f7793d3831cc70952383bbf2c9307d41394b1839.